docs/content/ingestion/firehose.md

```diff
@@ -27,6 +27,31 @@ This firehose acts as a Kafka consumer and ingests data from Kafka.
 
 This firehose ingests events from a predefined list of S3 objects.
 
+#### StaticAzureBlobStoreFirehose
```

Comment:
Sure. Do you think it would make sense to include an example of what a file might look like?

```json
{"foo": "bar", "baz": 5}
{"foo": "qux", "baz": 15}
{"foo": "blah", "baz": 11}
```